src/Http/ResponseEmitter.php

```diff
@@ -267,9 +267,10 @@ protected function emitCookies(array $cookies): void
                 $key = strtolower($key);
                 $data[$key] = $value;
             }
-            if (!empty($data['expires'])) {
+            if (is_string($data['expires'])) {
```

Comment:
This is wrong and a hard BC break!
It can be an integer, as also seen in line 252 default value - and often people use that for the argument to strtotime() as well.

Reply:
If the values is already integer then it doesn't need `strtotime()` conversion does it?  It's argument is supposed to be a string.

Reply:
So it seems if we fix the default value here in line 252 to null it could be also fine.
The cases for integer here do not seem valid.